node_scripts/install/spark.py

```diff
@@ -63,17 +64,17 @@ def generate_jupyter_config():
         display_name="PySpark",
         language="python",
         argv=[
-            "/usr/bin/python3",
+            "python",
             "-m",
             "ipykernel",
             "-f",
             "{connection_file}",
         ],
         env=dict(
             SPARK_HOME=spark_home,
-            PYSPARK_PYTHON="/usr/bin/python3",
-            PYSPARK_SUBMIT_ARGS="--master spark://{0}:7077 pyspark-shell".format(
-                master_node_ip),
+            PYSPARK_PYTHON="python",
```

Comment:
so wait is it using the docker file PYSPARK_PYTHON env at all. we should be able to remove one or the other

Reply:
oh yeah. they are the same thing actually... so i should just just the env var probably

Reply:
actually, i take back this comment - i think we should actually just python instead of the env var... because if you select python3.5.4 on your docker image, you would pretty much expect that jupyter will also run with pyspark python version 3.5.4... i can't imagine a case where that isn't true.